tests/e2e/test_rosacli_operator_roles.go

```diff
@@ -381,3 +386,335 @@ var _ = Describe("Edit operator roles", labels.Feature.OperatorRoles, func() {
+				Expect(err).To(BeNil())
+				CD, err := clusterService.ReflectClusterDescription(output)
+				Expect(err).To(BeNil())
+				operatorRolesArns := CD.OperatorIAMRoles
+				for _, policyArn := range operatorRolesArns {
+					_, operatorRoleName, err := common.ParseRoleARN(policyArn)
+					Expect(err).To(BeNil())
+					attachedPolicy, err := awsClient.ListAttachedRolePolicies(operatorRoleName)
+					Expect(err).To(BeNil())
+					rolePolicyMap[operatorRoleName] = *attachedPolicy[0].PolicyArn
+				}
+
+				By("Update opertor-roles policies permission")
+				updatedPolicyDocument := `{
+					"Version": "2012-10-17",
+					"Statement": [
+						{
+							"Action": [
+								"autoscaling:DescribeAutoScalingGroups"
+							],
+							"Effect": "Allow",
+							"Resource": "*"
+						}
+					]
+				}`
+				for _, policyArn := range rolePolicyMap {
+					_, err = awsClient.IamClient.CreatePolicyVersion(context.Background(), &iam.CreatePolicyVersionInput{
+						PolicyArn:      aws.String(policyArn),
+						PolicyDocument: aws.String(updatedPolicyDocument),
+						SetAsDefault:   true,
+					})
+					Expect(err).To(BeNil())
+				}
+
+				By("Detach and Delete one operator-role policy,openshift-cluster-csi-drivers-ebs")
+				var deletingPolicyRoleName string
+				for roleName := range rolePolicyMap {
+					if strings.Contains(roleName, "openshift-cluster-csi-drivers-ebs") {
+						deletingPolicyRoleName = roleName
+						break
+					}
+				}
+				err = awsClient.DetachRolePolicies(deletingPolicyRoleName)
+				Expect(err).To(BeNil())
+				err = awsClient.DeleteIAMPolicy(rolePolicyMap[deletingPolicyRoleName])
+				Expect(err).To(BeNil())
+
+				By("Create operator-role forcely")
+				output, err = ocmResourceService.CreateOperatorRoles(
+					"-c", clusterID,
+					"--mode", "auto",
+					"--force-policy-creation",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("Created role"))
+
+				By("Check the operator role policies are regenerated")
+
+				for roleName, policy := range rolePolicyMap {
+					policy, err := awsClient.GetIAMPolicy(policy)
+					Expect(err).To(BeNil())
+					policyVersion, err := awsClient.IamClient.GetPolicyVersion(context.TODO(), &iam.GetPolicyVersionInput{
+						PolicyArn: aws.String(*policy.Arn),
+						VersionId: policy.DefaultVersionId,
+					})
+					Expect(err).To(BeNil())
+					for rolSubName, permission := range operatorRoleNamePermissionMap {
+						if strings.Contains(roleName, rolSubName) {
+							Expect(*policyVersion.PolicyVersion.Document).To(ContainSubstring(permission))
+						}
+					}
+				}
+			})
+	})
+var _ = Describe("create IAM roles forcely testing",
+	func() {
+		defer GinkgoRecover()
+		var (
+			rosaClient          *rosacli.Client
+			ocmResourceService  rosacli.OCMResourceService
+			awsClient           *aws_client.AWSClient
+			err                 error
+			accountRolePrefix   = "ar59551"
```

Comment:
Can we just clarify the accountRolePrefix here and put value in the cases? If it is fixed and in future there are new cases added it will cause confusion